controllers/slicegateway/slicegateway.go

```diff
@@ -599,7 +599,7 @@ func (r *SliceGwReconciler) ReconcileGwPodStatus(ctx context.Context, slicegatew
 	return ctrl.Result{}, nil, false
 }
 
-func (r *SliceGwReconciler) SendConnectionContextToGwPod(ctx context.Context, slicegateway *kubeslicev1beta1.SliceGateway) (ctrl.Result, error, bool) {
+func (r *SliceGwReconciler) SendConnectionContextToGwPodAndQos(ctx context.Context, slice *kubeslicev1beta1.Slice, slicegateway *kubeslicev1beta1.SliceGateway) (ctrl.Result, error, bool) {
```

Comment:
nit: the function name could be more appropriate
SendConnectionContextAndQosToGwPod